initrd/etc/functions

```diff
@@ -122,6 +122,12 @@ enable_usb()
 		|| die "xhci_pci: module load failed"
 		sleep 2
 	fi
+
+	if [ $CONFIG_USB_KEYBOARD" = "y" ]; then
+		insmod /lib/modules/usbhid.ko \
+		|| die "usbhid: module load failed"
```

Comment:
```suggestion
	if [ "$CONFIG_USB_KEYBOARD" = "y" ]; then
		insmod /lib/modules/usbhid.ko ||
		die "usbhid: module load failed"
```

Reply:
Ah, does Heads disable autoloading?

Reply:
@DemiMarie Its litterally the other way around: there is no autoloading :)
The modules needs to be loaded in the right order to be loaded here.
CONFIG_LINUX_USB_COMPANION_CONTROLLER is old USB1 (only required per KGPE-D16 boards configs)
https://github.com/osresearch/heads/blob/714dbd7ac3790322166df67fe317b170ab4e33ee/initrd/etc/functions#L92-L133

Then when usb storage is needed, the module dependencies (enable_usb) is called just right before. 
https://github.com/osresearch/heads/blob/182bd6bd7517c8e322bc4712db01906b41cce58c/initrd/bin/mount-usb#L11

Reply:
Thanks for the code review.

Reply:
You’re welcome!

Reply:
@DemiMarie : this is the wrapper to insmod I was talking about earlier: https://github.com/osresearch/heads/blob/51ecbdc8cb7e280744c6a00eb3a8b2a1f116c1be/initrd/sbin/insmod#L1-L46

Where going to the recovery shell (exiting menu guided interactions) scrambles the PCRs, preventing secrets to be unsealed: https://github.com/osresearch/heads/blob/182bd6bd7517c8e322bc4712db01906b41cce58c/initrd/etc/functions#L13